src/tests/test_util.py

```diff
@@ -143,3 +144,44 @@ def test_parse_sql_results_to_json(ip, capsys, rows, columns, expected_json):
 def test_is_sqlalchemy_error(string, substrings, expected):
     result = util.if_substring_exists(string, substrings)
     assert result == expected
+
+
+@pytest.mark.parametrize(
+    "args, cmd_from, expected_error_message",
+    [
+        (
+            ["--table", "--table"],
+            "sqlcmd",
+            "Duplicate arguments in %sqlcmd. \
+Please use only one of each of the following: --table",
+        ),
+        (
+            ["--alias", "--alias"],
+            "sql",
+            "Duplicate arguments in %sql. \
+Please use only one of each of the following: --alias",
+        ),
+        (
+            ["--table", "--table", "--column", "--column"],
+            "sqlplot",
+            "Duplicate arguments in %sqlplot. \
+Please use only one of each of the following: --column, --table",
+        ),
+    ],
+)
+def test_check_duplicate_arguments_raises_usageerror(
+    args, cmd_from, expected_error_message
+):
+    with pytest.raises(UsageError) as excinfo:
+        util.check_duplicate_arguments(args, cmd_from)
```

Comment:
We should add test cases for `%sqlplot` command also apart from unit cases for `check_duplicate_arguments`.
Also, please add mixed cases like `--table --column --column`.
Plot specific cases like `--binwidth` etc.

Reply:
Added cases [here](https://github.com/ploomber/jupysql/pull/908/files#diff-6693dbe195b58d279ef47166dd1c83c4fe8153dd95a1c1521330f0e2d60033c2R322-R452)